PasteBinService/RegisterService/src/main/java/org/potrohit/springboot/RegisterService/activity/common/RegisterPasteBinRequest.java

```diff
@@ -0,0 +1,13 @@
+package org.potrohit.springboot.RegisterService.activity.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
@data comes with f @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor. 

So remove line 8,9 for all classes. 

Instead add @builder and try to build objectes using.